aries_cloudagent/wallet/routes.py

```diff
@@ -498,9 +498,12 @@ async def wallet_set_public_did(request: web.BaseRequest):
     )
     routing_keys = None
     mediator_endpoint = None
```

Comment:
These two lines should no longer be necessary

Reply:
Sorry, minor nitpick -- I should add that these changes look pretty good otherwise :slightly_smiling_face:

Reply:
Sorry, totally missed those, I will fix it right away :)